bundles/SeoBundle/src/Installer.php

```diff
@@ -47,10 +47,13 @@ private function addUserPermission(): void
         $db = \Pimcore\Db::get();
 
         foreach (self::USER_PERMISSIONS as $permission) {
-            $db->insert('users_permission_definitions', [
-                $db->quoteIdentifier('key') => $permission,
-                $db->quoteIdentifier('category') => self::USER_PERMISSIONS_CATEGORY,
-            ]);
+            $permissionExists = $db->fetchOne('SELECT count(*) FROM users_permission_definitions');
```

Comment:
You shouldn't fetch inside a loop!

Reply:
You completely right. If this was a performance critical piece of code that was running  several times a second day out and day in that would have been a no-no.
But remember the context here: You do inserts in the loop, a lot costlier than fetch, and it's three values, and it's done one time per bundle install. Surely that must be legitimately insignificant for this specific case?

Reply:
famous last words :P